english/play-music/playmusic.md

```diff
@@ -1 +1,3 @@
 You decide to just [Play some music](https://youtu.be/IcrbM1l_BoI)
+
+But some time after, you get tired of hearing the same song, that's why you decide to listen to ["The strokes"](theStrokes/theStrokes.md) or something more popular as the music for [Colombian people](colombianMusic/colombianMusic.md).
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.